openshift-install-powervs

```diff
@@ -31,6 +31,7 @@ Available commands:
   variables       Interactive way to populate the variables file
   create          Create an OpenShift cluster
   destroy         Destroy an OpenShift cluster
+  upgrade         Upgrade an OpenShift cluster
```

Comment:
Why we need to do upgrade using this script and not by simply following the available instructions and using `oc` or the web ui ?